joomla/joomla-cms
# [5.3] Form control fields removable token

Pull Request for Issue # .

### Summary of Changes

I had a request to make form token removable for ["Form control fields API" ](https://github.com/joomla/joomla-cms/pull/43857). 
So here it is.


### Testing Instructions

Edit https://github.com/joomla/joomla-cms/blob/b9820f1fed5f2bb98611c29fc68aae1f8be8ad28/administrator/components/com_content/src/View/Article/HtmlView.php#L136-L140
And add :
```php
$this->form->removeControlField('joomla.form.token');
```

Open Article editing form, and check HTML for the from. 
The token should be removed.

### Actual result BEFORE applying this Pull Request
Field with token is always rendered.
![Screenshot 2024-10-27_11-37-48](https://github.com/user-attachments/assets/bb0f397f-f5ea-423a-bd7e-24b820072620)

### Expected result AFTER applying this Pull Request
Field with token is removed.
![Screenshot 2024-10-27_11-37-11](https://github.com/user-attachments/assets/26cabfad-53aa-4d57-82a4-a824313af6d6)


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/322

Reference:
- https://github.com/joomla/joomla-cms/pull/43857
